internal/bminventory/inventory.go

```diff
@@ -2077,6 +2095,42 @@ func (b *bareMetalInventory) processDhcpAllocationResponse(ctx context.Context,
 	return b.clusterApi.SetVipsData(ctx, &cluster, apiVip, ingressVip, dhcpAllocationReponse.APIVipLease, dhcpAllocationReponse.IngressVipLease, b.db)
 }
 
+func (b *bareMetalInventory) processNtpSynchronizerResponse(ctx context.Context, host *models.Host, ntpSynchronizerResponseStr string) error {
+	var (
+		err                     error
+		ntpSynchronizerResponse models.NtpSynchronizationResponse
+	)
+
+	log := logutil.FromContext(ctx, b.log)
+
+	if err = json.Unmarshal([]byte(ntpSynchronizerResponseStr), &ntpSynchronizerResponse); err != nil {
+		log.WithError(err).Warnf("Json unmarshal ntp synchronizer response from host %s", host.ID.String())
+		return err
+	}
+
+	if len(ntpSynchronizerResponse.NtpSources) == 0 {
+		err = errors.Errorf("No NTP sources for host %s", host.ID.String())
+		log.WithError(err).Warnf("Update NTP sources")
+		return err
+	}
+
+	bytes, err := json.Marshal(ntpSynchronizerResponse.NtpSources)
+	if err != nil {
+		log.WithError(err).Warnf("Failed to marshal NTP sources for host %s", host.ID.String())
+		return err
+	}
+
+	if err = b.db.Model(&models.Host{}).Where("id = ? and cluster_id = ?", host.ID.String(),
```

Comment:
Maybe it should be part of host API? `SetNTP` or something like this.  the same as `UpdateInventory`